iothub/device/src/DirectMethod/DirectMethodResponse.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices.Client
+{
+    /// <summary>
+    /// The device/module's response to a direct method invocation.
+    /// </summary>
+    public class DirectMethodResponse
+    {
+        /// <summary>
+        /// Initializes an instance of this class.
+        /// </summary>
+        public DirectMethodResponse()
+        {
+        }
+
+        /// <summary>
+        /// The status of direct method response.
+        /// </summary>
+        [JsonProperty("status")]
+        public int Status { get; set; }
+
+        /// <summary>
+        /// The optional direct method payload.
+        /// </summary>
+        [JsonIgnore]
+        public byte[] Payload { get; set; }
+
+        /// <summary>
+        /// The request Id for the transport layer.
+        /// </summary>
+        internal string RequestId { get; set; }
```

Comment:
No JsonProperty?

Reply:
This field is set in the topic string over MQTT or in properties over AMQP, so it isn't part of the serialized payload

Reply:
Could you add JsonIgnore attribute and include a code comment to this effect?

Reply:
Sure thing